src/components/Icon/Icon.scss

```diff
@@ -27,192 +27,366 @@
 }
 
 .isColored {
-  color: color('white');
+  color: var(--p-surface, color('white'));
 }
 
 .colorWhite {
   @include color-icon('white');
   color: transparent;
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon-subdued));
+  }
 }
 
 .colorBlack {
   @include color-icon('black');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorSkyLighter {
   @include color-icon('sky', 'lighter');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon-subdued));
+  }
 }
 
 .colorSkyLight {
   @include color-icon('sky', 'light');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon-subdued));
+  }
 }
 
 .colorSky {
   @include color-icon('sky');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon-subdued));
+  }
 }
 
 .colorSkyDark {
   @include color-icon('sky', 'dark');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon-subdued));
+  }
 }
 
 .colorInkLightest {
   @include color-icon('ink', 'lightest');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorInkLighter {
   @include color-icon('ink', 'lighter');
 
   &::before {
-    background-color: color('sky');
+    background-color: var(--p-surface, color('sky'));
+  }
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
   }
 }
 
 .colorInkLight {
   @include color-icon('ink', 'light');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorInk {
   @include color-icon('ink');
 
   &::before {
-    background-color: color('sky');
+    background-color: var(--p-surface, color('sky'));
+  }
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
   }
 }
 
 .colorBlueLighter {
   @include color-icon('blue', 'lighter');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorBlueLight {
   @include color-icon('blue', 'light');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorBlue {
   @include color-icon('blue');
+
+  &.newDesignLanguage {
+    @include recolor-icon(var(--p-icon));
+  }
 }
 
 .colorBlueDark {
   @include color-icon('blue', 'dark');
 
-  &::before {
-    background-color: color('blue', 'light');
```

Comment:
Forgot to write a note about this earlier. This value is not a supported backdrop, and must have been missed in previous changes